apps/write-freely/metadata/description.md

```
@@ -0,0 +1,71 @@
+&nbsp;
+<p align="center">
+	<a href="https://writefreely.org"><img src="https://writefreely.org/img/writefreely.svg" width="350px" alt="WriteFreely" /></a>
+</p>
+<hr />
+&nbsp;
+
+WriteFreely is a clean, minimalist publishing platform made for writers. Start a blog, share knowledge within your organization, or build a community around the shared act of writing.
+
+![](https://i.snap.as/cQZxXoz.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)